LiskHQ/lisk-sdk
# Fix getTransactions endpoint expectation - Closes #5484

### What was the problem?

This PR resolves #5484 

### How was it solved?

- Fix the test expectation
- Update schema to allow empty query

### How was it tested?

- Link with lisk-core
- Start 2 nodes, one forging and one syncing
- Observe syncing to each other
